src/presentation/index.ts

```diff
@@ -1,12 +1,13 @@
 import cors from 'cors'
-import express, { Express, Router } from 'express'
-import { BaseController } from './controllers/base-controller'
-import { ControllerFacade } from './facades/controller-facade'
-import { EventServerFacade } from './facades/event-server-facade'
-import { ServiceFacade } from '../business-logic/facades/service-facade'
-import { Logger } from '../logger/logger'
-import { LoggerFacade } from '../logger/logger-facade'
-import { RepositoryFacade } from '../data-access/facades/repository-facade'
+import express, {Express, NextFunction, Request, Response, Router} from 'express'
+import {BaseController} from './controllers/base-controller'
+import {ControllerFacade} from './facades/controller-facade'
+import {EventServerFacade} from './facades/event-server-facade'
+import {ServiceFacade} from '../business-logic/facades/service-facade'
+import {Logger} from '../logger/logger'
+import {LoggerFacade} from '../logger/logger-facade'
+import {RepositoryFacade} from '../data-access/facades/repository-facade'
+import bodyparser from 'body-parser'
```

Comment:
Minor thing: Since body parser is two words, the import should use camelCasing: `bodyParser` :)